library/Ivoz/Provider/Domain/Model/Administrator/Administrator.php

```diff
@@ -120,6 +120,10 @@ public function getTimezone()
             return $timeZone;
         }
 
+        if (!$this->getCompany()) {
+            return null;
+        }
+
```

Comment:
I like this condition the other way arround, because will make a brand timezone condition easier to include.
```
if ($this->getBrand()) {
   return BrandTimeZone.
}


if ($this->getCompany()) {
    return CompanyTimeZone.
}

return null
```